src/main/resources/application.properties

```diff
@@ -14,15 +14,16 @@ spring.datasource.password=
 
 ## Spring Liquibase
 spring.liquibase.change-log=classpath:liquibase/db.changelog-master.xml
-spring.liquibase.url=jdbc:mysql://localhost:3306/evo_exchange?serverTimezone=UTC&createDatabaseIfNotExist=true&useSSL=false
-spring.liquibase.user=root
-spring.liquibase.password=
 
 ### Hibernate Properties
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=update
 
+## Spring View Resolver
+spring.mvc.view.prefix=html/
```

Comment:
I don't see the same configuration for test purpuses.

Reply:
Performed.